test/Test/Stan/Inspection.hs

```diff
@@ -12,9 +12,16 @@ import qualified Stan.Inspection.Partial as Stan
 
 inspectionsSpec :: Spec
 inspectionsSpec = describe "Inspections by ID" $ do
-    describe "Partial" $
+    describe "Partial" $ do
         it "STAN-0001 should be partial head" $
             getInspectionById Stan.stan0001 `shouldBe` Stan.stan0001Inspection
+        it "STAN-0002 should be partial tail" $
+            getInspectionById Stan.stan0002 `shouldBe` Stan.stan0002Inspection
+        it "STAN-0003 should be partial init" $
+            getInspectionById Stan.stan0003 `shouldBe` Stan.stan0003Inspection
+        it "STAN-0004 should be partial last" $
+            getInspectionById Stan.stan0004 `shouldBe` Stan.stan0004Inspection
```

Comment:
I wonder, if we can automate at least this set of tests :thinking: Something like:

```haskell
forM_ (zip partialInspections partialInspectionsIds) $ \(ins, insId) ->
    it (unId insId <> ": " <> inspectionName ins) $
        getInspectionById insId `shouldBe` ins
```